publish/2019-11-20-WhatsNew030.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "What's New in Release 0.3.0"
+categories: blog
+author: Stephen Kinder
+author_picture: https://avatars3.githubusercontent.com/u/50876789
+author_github: https://github.com/stephenkinder
+additional_authors: 
+ - name: Nate Ziemann
+   github: https://github.com/nateziemann
+   image: https://avatars1.githubusercontent.com/u/28736793?s=400&v=4
+seo-title: What is new in release 0.3.0
+seo-description: See what is hot in Kabanero v0.3.0.
+blog_description: "See what is hot in Kabanero v0.3.0."
+---
+
+Today you can get your hands on the next monthly release of Kabanero, v0.3.0.  Check out all the details on everything new below include a shift from OKD 3.11 to OpenShift 4.2 and an integrated Hosted IDE.  
+
+Hosted IDE with Eclipse Che and Eclipse Codewind:
+Expanding on microservice development scenarios, Kabanero can now establish an Eclipse Che environment. Developers can remotely leverage Codewind in Che, hosted in the same Kubernetes cluster as the Kabanero Instance, to develop microservices. This capability reduces the computing resources that are needed for individual developers.
+
+OpenShift 4.2 support:
+Kabanero is moving very rapidly. To accommodate the desired rate of innovation, especially around Kubernetes operators, we decided to move our foundation from OKD 3.11 to OpenShift Container Platform 4.2.  While our community is committed to being open, the OKD distribution is currently behind the commercial distribution.  Once OKD 4.x is available we will shift our recommendation for trying Kabanero by deploying on OKD. 
```

Comment:
Add comma and suggest from/to?

Once OKD 4.x is available, we will shift our recommendation for trying Kabanero *to* deploying on OKD.